rules/order/tests/Rector/ClassLike/OrderMethodsByVisibilityRector/Fixture/abstact.php.inc

```diff
@@ -20,9 +36,21 @@ namespace Rector\Order\Tests\Rector\ClassLike\OrderMethodsByVisibilityRector\Fix
 abstract class AbstractClass
 {
     abstract public function abstractPublicFunctionName();
+    public function publicFunctionName()
+    {
+
+    }
     abstract static public function abstractStaticPublicFunctionName();
     abstract protected function abstractProtectedFunctionName();
+    protected function protectedFunctionName()
+    {
+
+    }
     abstract static protected function abstractStaticProtectedFunctionName();
+    private function privateFunctionName()
+    {
+
+    }
```

Comment:
@TomasVotruba does this order for abstract classes make sense to you? Basically, abstract methods are in the beginning, like this:
```php
    abstract public function abstractPublicFunctionName();
    public function publicFunctionName()
    final public function finalPublicFunctionName()
    abstract static public function abstractStaticPublicFunctionName();
    static public function staticPublicFunctionName()
    static final public function staticFinalPublicFunctionName()

    abstract protected function abstractProtectedFunctionName();
    protected function protectedFunctionName()
    final protected function finalProtectedFunctionName()
    abstract static protected function abstractStaticProtectedFunctionName();
    static protected function staticProtectedFunctionName()
    static final protected function staticFinalProtectedFunctionName()

    private function privateFunctionName()
    final private function finalPrivateFunctionName()
    static private function staticPrivateFunctionName()
    static final private function staticFinalPrivateFunctionName()
```

Reply:
If it makes and the PR looks good to you, it can be merged.